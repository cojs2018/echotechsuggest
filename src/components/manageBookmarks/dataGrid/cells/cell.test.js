import React from 'react';
import render from 'react-test-renderer';

import Cell from "./cell";

describe('Cell', () => {
    it('renders correctly', () => {
        const props = {
            value: 'Test Cell'
        }

        const tree = render
            .create(<Cell {...props} />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})