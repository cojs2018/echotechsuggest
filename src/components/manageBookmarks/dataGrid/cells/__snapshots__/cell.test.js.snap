// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      false,
      Array [
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
        },
        undefined,
        undefined,
      ],
    ]
  }
>
  <Text
    numberOfLines={1}
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "System",
          "fontWeight": "400",
          "textAlign": "left",
        },
        undefined,
      ]
    }
  >
    Test Cell
  </Text>
</View>
`;